sharedUiComponents/lines/sliderLineComponent.tsx

```diff
@@ -111,9 +111,11 @@ export class SliderLineComponent extends React.Component<ISliderLineComponentPro
         return (
             <div className="sliderLine">
                 {this.props.icon && <img src={this.props.icon} className="icon"/>}
-                <div className={this.props.margin ? "label withMargins" : "label"}  title={this.props.label}>
-                    {this.props.label}
-                </div>
+                {(!this.props.icon || (this.props.icon && this.props.label != "")) &&

```

Comment:
```suggestion
                {(!this.props.icon ||  this.props.label !== "") &&
```
No need to check if `this.props.icon` exists, it was established already (by checking if it doesn't exist)